cli/kata-env.go

```diff
@@ -276,6 +282,7 @@ func getHypervisorInfo(config oci.RuntimeConfig) HypervisorInfo {
 		Path:              hypervisorPath,
 		BlockDeviceDriver: config.HypervisorConfig.BlockDeviceDriver,
 		Msize9p:           config.HypervisorConfig.Msize9p,
+		UseVSock:          config.HypervisorConfig.UseVSock,
```

Comment:
Nice. But please can you bump `formatVersion` as this changes the output format of `kata-env`.

Reply:
done

Reply:
Ta.